### DISCUSSION TITLE
Welcome to iptablesUtils Discussions!

### DISCUSSION THREAD

**Discussion by arloor** [arloor/iptablesUtils#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JackPumkinKing** [comment]
Why do SUBNET caps only go up to 255.  Also is it worth the 003023 I  this day and age to  use DCHP cycling when we had access to free VPNS like proton VPN?
